### DISCUSSION TITLE
Not all languages are being displayed

### DISCUSSION THREAD

**Discussion by kalebers** [anuraghazra/github-readme-stats#3635] [Q&A] [answered]
### Describe the bug

I have more languages in repos that aren't being displayed on my status (including  SQL and Powershell bash), even setting up the number of languages on display. 

### Expected behavior

Languages included on repos should be displayed on status.

**Comment by qwerty541** [comment, marked as answer]
Hey, @kalebers! I don't see any your repository which includes sql or bash code, so I assume that it is your private contributions. By default top languages card shows data only from public non-forked repositories. To include languages used in private repositories, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token. Our documentation contains that info.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/61ca6142-db4a-43af-9348-a7b7ea8fefdc)
